mitodl/open-discussions
# Podcast / PodcastEpisode drawer support

#### Pre-Flight checklist



#### What are the relevant tickets?

part of #2722 

#### What's this PR do?

adds basic support for podcasts and podcast episodes to our learning resource drawer.

#### How should this be manually tested?

you should be able to open any podcast or podcast episode in the drawer, the display should be adequate. you should be able to start playing an episode from the drawer.

#### Screenshots (if appropriate)

![Screenshot from 2020-04-27 11-17-38](https://user-images.githubusercontent.com/6207644/80389168-be658a00-8878-11ea-9ea4-6967dcab69c1.png)
![Screenshot from 2020-04-27 11-17-29](https://user-images.githubusercontent.com/6207644/80389170-be658a00-8878-11ea-8351-f9244aa89184.png)
![Screenshot from 2020-04-27 11-16-49](https://user-images.githubusercontent.com/6207644/80389172-befe2080-8878-11ea-9655-0cc125fec47b.png)
![Screenshot from 2020-04-27 11-16-38](https://user-images.githubusercontent.com/6207644/80389175-befe2080-8878-11ea-8bd6-24dfc09df4e6.png)


#### What GIF best describes this PR or how it makes you feel?
(Optional)
